packages/yourloops/components/menus/team-scope-menu.tsx

```diff
@@ -139,7 +142,7 @@ export const TeamScopeMenu: FunctionComponent = () => {
 
   return (
     <>
-      <Box maxWidth={MENU_MAX_WIDTH_PX}>
+      <Box>
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏I think that this box is no longer useful